client/core/trade.go

```diff
@@ -1239,14 +1245,120 @@ func (t *trackedTrade) isSwappable(ctx context.Context, match *matchTracker) (re
 				"but we have not seen the counter-party's redemption yet. This could just"+
 				" be network latency.", match)
 		}
+
 		match.setSwapConfirms(int64(confs))
+
 		t.notify(newMatchNote(TopicConfirms, "", "", db.Data, t, match))
 		return false, false
 	}
 
 	return false, false
 }
 
+// updateDynamicSwapFeesPaid starts a goroutine that updates the fees used for
```

Comment:
> starts a goroutine

Do we have to? This seems like it would be simpler and more on-pattern without a goroutine.

Reply:
I'll see what it looks like.

Reply:
Not a goroutine https://github.com/decred/dcrdex/compare/0f8028f67ae1b69f3bf1563dce2027543d06322c..e411b4e266045aa22009185c9b6bede40b02bc58